README.md

```diff
@@ -1,2 +1,59 @@
 # Salesforce-Postman
 Salesforce API Postman Collection
```

Comment:
```
A collection of [Postman](https://www.postman.com/) resources for the Salesforce APIs:
<list APIs here>
```
Add a single screenshot of Postman with part of the collection expanded and a request with a response.